src/main/java/core/mate/academy/service/BulldozerProducerImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducerImpl implements MachineProducer<Bulldozer> {

```

Comment:
Don't begin class or method implementation with an empty line. Remove the empty line at the start of the class definition.